conf/.env

```diff
@@ -0,0 +1,378 @@
+# example :  LDAP_SEARCH_PAGE_SIZE=12345
+#LDAP_SEARCH_PAGE_SIZE=0
+LDAP_SEARCH_PAGE_SIZE=0
+
+# LDAP_SEARCH_SIZE_LIMIT : The limit number of entries (0=unlimited)
+# example :  LDAP_SEARCH_SIZE_LIMIT=12345
+#LDAP_SEARCH_SIZE_LIMIT=0
+LDAP_SEARCH_SIZE_LIMIT=0
+
+# LDAP_GROUP_FILTER_ENABLE : Enable group filtering
+# example :  LDAP_GROUP_FILTER_ENABLE=true
+#LDAP_GROUP_FILTER_ENABLE=false
+LDAP_GROUP_FILTER_ENABLE=true
+
+# LDAP_GROUP_FILTER_OBJECTCLASS : The object class for filtering
+# example :  LDAP_GROUP_FILTER_OBJECTCLASS=group
+#LDAP_GROUP_FILTER_OBJECTCLASS=
+LDAP_GROUP_FILTER_OBJECTCLASS=posixGroup
+
+# LDAP_GROUP_FILTER_GROUP_ID_ATTRIBUTE :
+# example :
+#LDAP_GROUP_FILTER_GROUP_ID_ATTRIBUTE=
+LDAP_GROUP_FILTER_GROUP_ID_ATTRIBUTE=cn
+
+# LDAP_GROUP_FILTER_GROUP_MEMBER_ATTRIBUTE :
+# example :
+#LDAP_GROUP_FILTER_GROUP_MEMBER_ATTRIBUTE=
+LDAP_GROUP_FILTER_GROUP_MEMBER_ATTRIBUTE=memberUid
+
+# LDAP_GROUP_FILTER_GROUP_MEMBER_FORMAT :
+# example :
+#LDAP_GROUP_FILTER_GROUP_MEMBER_FORMAT=
+LDAP_GROUP_FILTER_GROUP_MEMBER_FORMAT=uid
+
+# LDAP_GROUP_FILTER_GROUP_NAME :
+# example : LDAP_GROUP_FILTER_GROUP_NAME=wekan_user
+#LDAP_GROUP_FILTER_GROUP_NAME=
+LDAP_GROUP_FILTER_GROUP_NAME=sftpusers
+
+# LDAP_UNIQUE_IDENTIFIER_FIELD : This field is sometimes class GUID (Globally Unique Identifier)
+# example :  LDAP_UNIQUE_IDENTIFIER_FIELD=guid
+#LDAP_UNIQUE_IDENTIFIER_FIELD=
+LDAP_UNIQUE_IDENTIFIER_FIELD=entryUUID
+
+# LDAP_UTF8_NAMES_SLUGIFY : Convert the username to utf8
+# example :  LDAP_UTF8_NAMES_SLUGIFY=false
+#LDAP_UTF8_NAMES_SLUGIFY=true
+LDAP_UTF8_NAMES_SLUGIFY=true
+
+# LDAP_USERNAME_FIELD : Which field contains the ldap username
+# example :  LDAP_USERNAME_FIELD=username
+#LDAP_USERNAME_FIELD=
+LDAP_USERNAME_FIELD=uid
+
+# LDAP_FULLNAME_FIELD : Which field contains the ldap fullname
+# example :  LDAP_FULLNAME_FIELD=fullname
+#LDAP_FULLNAME_FIELD=
+LDAP_FULLNAME_FIELD=cn
+
+# LDAP_MERGE_EXISTING_USERS :
+# example :  LDAP_MERGE_EXISTING_USERS=true
+#LDAP_MERGE_EXISTING_USERS=false
+LDAP_MERGE_EXISTING_USERS=true
+
+# LDAP_EMAIL_MATCH_ENABLE : allow existing account matching by e-mail address when username does not match
+# example: LDAP_EMAIL_MATCH_ENABLE=true
+#LDAP_EMAIL_MATCH_ENABLE=false
+LDAP_EMAIL_MATCH_ENABLE=true
+
+# LDAP_EMAIL_MATCH_REQUIRE : require existing account matching by e-mail address when username does match
+# example: LDAP_EMAIL_MATCH_REQUIRE=true
+#LDAP_EMAIL_MATCH_REQUIRE=false
+LDAP_EMAIL_MATCH_REQUIRE=false
+
+# LDAP_EMAIL_MATCH_VERIFIED : require existing account email address to be verified for matching
+# example: LDAP_EMAIL_MATCH_VERIFIED=true
+#LDAP_EMAIL_MATCH_VERIFIED=false
+LDAP_EMAIL_MATCH_VERIFIED=false
+
+# LDAP_EMAIL_FIELD : which field contains the LDAP e-mail address
+# example: LDAP_EMAIL_FIELD=mail
+#LDAP_EMAIL_FIELD=
+LDAP_EMAIL_FIELD=mail
+
+# LDAP_SYNC_USER_DATA :
+# example :  LDAP_SYNC_USER_DATA=true
+#LDAP_SYNC_USER_DATA=false
+LDAP_SYNC_USER_DATA=true
```

Comment:
I am not sure it's a good idea to sync those info . What if the user change the email inside LDAP ?

Reply:
Seems it works differently, so it's ok